mobify/progressive-web-scaffold
# Add login form using FormFields component

**JIRA**: https://mobify.atlassian.net/browse/WEB-723
 **Linked PRs**: https://github.com/mobify/progressive-web-sdk/pull/183

## Changes
- Uses the FormFields component to build the form.

## How to test-drive this PR
- `npm run dev`
- Browse to the login page
- Make sure the login form works, and has proper validation
